### PR TITLE
chore: check for accepting users before joining a realm

### PR DESCRIPTION
**Pull request by hugoArregui**

### CODE REVIEW

File: packages/shared/dao/sagas.ts [hunk 1]
```diff
@@ -198,7 +198,8 @@ function* initializeCatalystCandidates() {
 
 export async function checkValidRealm(baseUrl: string): Promise<PingResult | null> {
   const pingResult = await ping(baseUrl + '/about')
-  if (pingResult.status === ServerConnectionStatus.OK) {
+  const acceptingUsers = pingResult.result && (pingResult.result.acceptingUsers ?? true)
```
Comment (menduz) [inline packages/shared/dao/sagas.ts:201]: Is this condition checked to return false when acceptingUsers == false?

Comment (hugoArregui) [reply packages/shared/dao/sagas.ts:201]: not sure to understand the question, sorry

Comment (kuruk-mm) [reply packages/shared/dao/sagas.ts:201]: I think the doubt comes from:
```ts
(pingResult.result.acceptingUsers ?? true)
```

but

```ts
(false ?? true) = false
```

so it should be ok

**Review by menduz — APPROVED** [review]

**Comment by menduz** [comment]
Approving, although this will give no hint about the realm being full. Maybe an exception can be considered to improve the UX

**Review by agusaldasoro — APPROVED** [review]

**Pull request merged**